cli/server_cmd.go

```diff
@@ -233,3 +232,17 @@ var serverCmd = &cobra.Command{
 		return nil
 	},
 }
+
+func loadAssets(manifestFile string) (map[string]string, error) {
+	data, err := os.ReadFile(manifestFile)
+	if err != nil {
+		return nil, fmt.Errorf("couldn't read manifest '%s': %w", manifestFile, err)
+	}
+
+	assets := make(map[string]string)
+	if err = json.Unmarshal(data, &assets); err != nil {
+		return nil, fmt.Errorf("couldn't parse manifest '%s': %w", manifestFile, err)
+	}
+
+	return assets, nil
+}
```

Comment:
This function might need to be placed somewhere else but didn't really know where.

Reply:
there's more stuff that needs a place outside the cli package in an application builder, we can move all it in a later refactor